DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,9 +16,10 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="29">
+  <class name="pat::Electron"  ClassVersion="30">
+   <version ClassVersion="30" checksum="3508125821"/>
    <version ClassVersion="29" checksum="1784986402"/>
-   <version ClassVersion="28" checksum="2518240031"/>
+   <version ClassVersion="28" checksum="1881133053"/>
```

Comment:
@gpetruc @Dr15Jones 
please confirm that the class version changes here make sense